Backend/lib/beckn-spec/src/Beckn/Types/Core/Taxi/Common/StartInfo.hs

```diff
@@ -16,14 +16,16 @@ module Beckn.Types.Core.Taxi.Common.StartInfo where
 
 import Beckn.Types.Core.Taxi.Common.Authorization
 import Beckn.Types.Core.Taxi.Common.Location (Location)
+import Beckn.Types.Core.Taxi.OnSearch (TimeTimestamp)
 import Data.OpenApi (ToSchema (..), defaultSchemaOptions)
 import EulerHS.Prelude hiding (id)
 import Kernel.Utils.JSON (removeNullFields)
 import Kernel.Utils.Schema (genericDeclareUnNamedSchema)
 
 data StartInfo = StartInfo
   { location :: Location,
-    authorization :: Maybe Authorization
+    authorization :: Maybe Authorization,
+    time :: TimeTimestamp
```

Comment:
Is this backward compatible changes? I mean if we deploy old version of bpp and new version of bap, then bpp will not send this field, and parsing on bap side will fail. Lets make it optional

Reply:
Sure